src/lib/openapi/spec/event-search-response-schema.ts

```diff
@@ -0,0 +1,34 @@
+import type { FromSchema } from 'json-schema-to-ts';
+import { eventSchema } from './event-schema';
+import { tagSchema } from './tag-schema';
+
+export const eventSearchResponseSchema = {
+    $id: '#/components/schemas/eventsSearchResponseSchema',
+    type: 'object',
+    additionalProperties: false,
+    required: ['events', 'total'],
+    description: 'A list of events that has happened in the system',
```

Comment:
Small suggestion here. The most important thing is changing "has" to "have".

```suggestion
    description: 'A list of events that have been registered by the system',
```